app/models/manageiq/providers/kubernetes/container_manager/scanning/job.rb

```diff
@@ -124,12 +124,24 @@ def analyze
                           msg % [options[:image_full_name], options[:docker_image_id][0..11], actual[0..11]],
                           'error')
     end
+
+    collect_compliance_data(image)
+
     image.scan_metadata(SCAN_CATEGORIES,
                         "taskid" => jobid,
                         "host"   => MiqServer.my_server,
                         "args"   => [YAML.dump(scan_args)])
   end
 
+  def collect_compliance_data(image)
+    # TODO: Collect openscap data once it is supported by image_inspector_client
+    #
+    # openscap_result = OpenscapResult.new(:container_image => image)
+    # openscap_result.raw = image_inspector_client.fetch_oscap_arf
+    #
+    # openscap_result.save
```

Comment:
@simon3z I thought I commented on this a while ago, saying the current implementation was OK.